### DISCUSSION TITLE
Top hole easily overblows to about 1 note high.

### DISCUSSION THREAD

**Discussion by skraninger** [edwardkort/WWIDesigner#162] [NAF Study]
Hi, thanks for creating this great program. I have made quite a few bamboo flutes using it but about 30% have an annoying overblow problem where all of the notes except the top note are fine, but the top note (actually the octave note where the all except the 3rd hole are open OOXOOO) overblows to a higher note very easily. It seems like the hole is not in a good place. Do you have any suggestions as to how to prevent this. I think I am doing all of the fipple calculations correctly.

**Comment by edwardkort** [comment]
Sounds like nodal interference: your top hole is too high up the bore. The following video describes how to address this:
http://youtu.be/F90l9vpMTZY

**Comment by bhp1** [comment]
Unfortunately, WIDesigner can model resonant frequencies, but can't always tell how solid the note will be when played.  You may be able to get some sense of it from the [Note Spectrum Graph](https://github.com/edwardkort/WWIDesigner/wiki/Reading-the-Note-Spectrum-Graph).  That page illustrates an apparent resonance (the one around 1950 Hz) that can't be played at all in practice.  

For the NAF study model, the graph doesn't show loop gain, but it does show the impedance ratio.  Predicted resonances are where the impedance ratio crosses zero with a positive slope.

**Reply by skraninger** [reply]
Thanks for the reponse.  I did review they nodal interference information, and it does seem to be the problem. Is there any reason that the nodal interference calculations are not included in widesinger?  I am a retired programmer and maybe could help to incorporate the calculations but would need to be pointed in the right direction (re the source code), unless this is really not feasible.

**Reply by edwardkort** [reply]
There are a few reasons why I didn't incorporate a calculation module for the top hole (nodal interference) ratio:
1. It has already been done well in Mike Prairie's spreadsheet.
2.  Since the calculation is not an optimization, it would need to added as a "wart" somewhere in the interface. I've resisted adding instrument-specific warts as much as possible.
3. The value of the ratio is fairly arbitrary. I know of some flute makers that always use a value of 0.33. For me, the calculated value standardly comes out at around 0.24 - so I use a ratio of 0.26 in WIDesigner, rarely doing the calculations.

**Comment by bhp1** [comment]
Flow/pressure nodal charts would help investigate this situation, as they are used in the Flutopedia page on [Nodal Interference](http://www.flutopedia.com/nodal_interference.htm).  I looked into this some time ago.  Unfortunately, the WIDesigner impedance calculation doesn't readily lend itself to generating these charts, so I didn't pursue the idea.
